Framework/Crystal/src/SetSpecialCoordinates.cpp

```diff
@@ -3,10 +3,15 @@
 #include "MantidAPI/IMDEventWorkspace.h"
```

Comment:
Did you change the base class for this algorithm? It should go to deprecatedalgorithm now.